source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,684 @@ Filters Builders
+   {"_id": 3, "color": "blue", "qty": 8, "vendor": ["B", "A"]}
+   {"_id": 4, "color": "white", "qty": 6, "vendor": ["D"]}
+   {"_id": 5, "color": "yellow", "qty": 11, "vendor": ["A", "B"]}
+   {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+   {"_id": 7, "color": "green", "qty": 8,"vendor": ["B", "C"]}
+   {"_id": 8, "color": "orange", "qty": 7, "vendor": ["A", "D"]}
+
+``binary_numbers``
+
+.. code-block:: json
+
+   { _id: 9, a: 54, binaryValue: "00110110" }
+   { _id: 10, a: 20, binaryValue: "00010100" }
+   { _id: 11, a: 68, binaryValue: "1000100" }
+   { _id: 12, a: 102, binaryValue: "01100110" }
+
+``geo_points``
+
+.. code-block:: json
+
+   { _id: 13, type: "Point", coordinates: [ 2,2 ] }
+   { _id: 14, type: "Point", coordinates: [ 5,6 ] }
+   { _id: 15, type: "Point", coordinates: [ 1,3 ] }
+   { _id: 16, type: "Point", coordinates: [ 4,7 ] }
+
+.. _comparison:
+
+Comparison 
+----------
+
+The comparison operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Comparison Operator
+     - Matches
+      
+   * - ``eq()``
+     - values that are equal to a specified value.
+
+   * - ``gt()``
+     - values that are greater than a specified value.
+      
+   * - ``gte()``
+     - values that are greater than or equal to a specified value.
+
+   * - ``lt()``
+     - values that are less than a specified value.
+
+   * - ``lte()``
+     - values that are less than or equal to a specified value.
+
+   * - ``ne()``
+     - values that are not equal to a specified value.
+
+   * - ``in()``
+     - any of the values specified in an array.
+
+   * - ``nin()``
+     - none of the values specified in an array.
+
+   * - ``empty()``
+     - all the documents.
+
+
+The following code snippet shows how to use a comparison filter:
+
+.. code-block:: java
+   
+   import static com.mongodb.client.model.Filters.*;
+
+   // <MongoCollection setup code here>
+
+   collection.find(<comparison filter>);
+
+.. example::
+
+   The following example creates a filter that selects all documents where
+   the value of the ``qty`` field equals ``5`` in the paint_purchaces collection:
+
+   .. code-block:: java
+      
+      import static com.mongodb.client.model.Filters.eq;
+
+      // <MongoCollection setup code here>
+
+      Bson equalComparison = eq("qty", 5);
+      List<Document> results = new ArrayList<>();
+      collection.find(equalComparison).into(results);
+      for (Document result : results) {
+         System.out.println(result.toJson());
+      }
+   
+   The Bson renders as:
+
+   .. code-block:: none
+      :copyable: false
+   
+      {
+         "qty": 20
+      }
+
+   The output of the above code snippet should look something like this:
+
+   .. code-block:: json
+      :copyable: false
+   
+      {"_id": 1, "color": "red", "qty": 5, "vendor": ["A"]}
+      {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+
+.. example::
```

Comment:
Is there a specific reason why you chose "empty" or can we choose something else? Since no filter is equivalent to the empty operator, and it's likely that no filter is used more often, I would use this space to demonstrate something else that might be used more often.

Reply:
The jira ticket requests for this method to be included

Reply:
Can we show an additional example, perhaps one of the filter builders that uses one or more parameters?

I don't mind keeping this example as well; the linked ticket wasn't specific as to whether to include an example or just the information you included in the table.